src/chain_spec.rs

```diff
@@ -67,11 +67,64 @@ pub fn local_testnet_config() -> ChainSpec {
 	)
 }
 
+pub fn breaknet4_config() -> ChainSpec {
```

Comment:
are you interested to remove this before master merge?

Reply:
I want to keep it on master at least for the next release. Most users only download release software, so we can get more people testing `breaknet4` before retiring it.